README.md

```diff
@@ -27,6 +27,7 @@ ShipFosdick.configure do |config|
   config.client_name = ENV.fetch('CLIENT_NAME')
   config.client_code = ENV.fetch('CLIENT_CODE')
   config.adcode = ENV.fetch('ADCODE')
+  config.shipment_updater_class = "ShipFosdick::IndividualShipmentUpdater"
```

Comment:
The main one I can think of is you can supply the classname in an environment variable. Is that a good enough reason? What do you think?